### ISSUE TITLE
Print facts to user while they wait for animation

### ISSUE THREAD

**Issue by collinsinclair** [collinsinclair/N-Body-Improved#6]
Including the patterns in the kinetic energies

**Comment by bkorb** [comment]
The current system interrupts the progress bar and could be improved by clearing the screen before each print statement.  This may make it hard to follow the story.  Additionally, after running several simulations the messages become repetitive.  Consider using a randomized set of brief facts instead of an ordered sequence of connected facts.

**Comment by collinsinclair** [comment]
Agreed on all points. I'll start cleaning this up
